docs/source/models.rst

```diff
@@ -195,19 +195,55 @@ Model Signature And Input Example
 ---------------------------------
 When working with ML models you often need to know some basic functional properties of the model
 at hand, such as "What inputs does it expect?" and "What output does it produce?". MLflow models can
-include the following additional metadata about model inputs and outputs that can be used by
+include the following additional metadata about model inputs, outputs and params that can be used by
 downstream tooling:
 
-* :ref:`Model Signature <model-signature>` - description of a model's inputs and outputs.
+* :ref:`Model Inference Params <inference-params>` - description of params used for model inference.
+* :ref:`Model Signature <model-signature>` - description of a model's inputs, outputs and parameters.
 * :ref:`Model Input Example <input-example>` - example of a valid model input.
 
+.. _inference-params:
+
+Model Inference Params
+^^^^^^^^^^^^^^^^^^^^^^
+Inference params are parameters that are passed to the model at inference time. These parameters
+do not need to be specified when training the model, but could be useful for inference. With the 
+advances in foundational models, more often "inference configuration" is used to modify the behavior 
+of a model. In some cases, especially popular LLMs, the same model may require different parameter 
+configurations for different samples. 
+
+With this newly introduced feature, you can now specify a dictionary of inference params during 
+model inference, and this could be extremely useful for LLMs. By passing different params such
+as ``temperature``, ``max_length``, etc. to the model at inference time, you can easily control 
+the output of the model.
+
+In order to use params at inference time, a valid :ref:`Model Signature <model-signature>` with 
+``params`` must be defined. The params are passed to the model at inference time as a dictionary,
+and each param value will be validated against the corresponding param type defined in the model
+signature. Valid param types are ``Union[DataType, List[DataType], None]`` where ``DataType`` is
```

Comment:
Instead of the typing signature here, could we just rely on the great bullet list that you have below and mention that it takes in either nothing (``None``), a ``DataType`` that is listed below, or a list of ``DataType``s that are listed below?